hydradx/model/amm/omnipool_amm.py

```diff
@@ -286,7 +286,8 @@ def __init__(self, state: OmnipoolState):
         # self.sub_pools = copy.deepcopy(state.sub_pools)
         self.oracles = {k: OracleArchiveState(v) for (k, v) in state.oracles.items()}
         self.unique_id = state.unique_id
-
+        self.volume_in = state.current_block.volume_in
```

Comment:
Are these changes an unrelated bug fix?

Reply:
No, this is so I can track trade volume as a percentage of the pool, because I think that's relevant to the returns LPs will get